src/components/PrimaryPanes/Outline.js

```diff
@@ -61,21 +61,17 @@ export class Outline extends Component<Props> {
     );
   }
 
-  renderClassFunctions(functions: SymbolDeclaration[]) {
-    const classFunctions = functions.filter(
-      func => func.name != "anonymous" && !!func.klass
-    );
-
-    if (classFunctions.length == 0) {
+  renderClassFunctions(klass: string, functions: SymbolDeclaration[]) {
+    if (klass == null || functions.length == 0) {
       return null;
     }
 
-    const klass = classFunctions[0].klass;
-    const klassFunc = functions.find(func => func.name === klass);
+    const classFunc = functions.find(func => func.name === klass);
```

Comment:
hehe `klass` is from my ruby days :) no fun!

Reply:
Ha! It makes sense here because `klass` is the preceding const but with that being moved it makes sense to change its spelling.